src/index.ts

```diff
@@ -1,81 +1,128 @@
+import * as logger from 'loglevel'
+
+interface LogMessage {
+  application: string
+  environment: string
+  level: string
+  message: string
+  timestamp: string
+}
+
 enum LogLevel {
-  INFO = 'info',
-  WARN = 'warn',
-  ERROR = 'error',
+  TRACE = 'TRACE',
+  DEBUG = 'DEBUG',
+  INFO = 'INFO',
+  WARN = 'WARN',
+  ERROR = 'ERROR',
+  SILENT = 'SILENT'
 }
 
-interface Message {
-  timestamp: string
-  log_level: string
-  event: string
-  body: unknown
-  message: {
-    event: string
+class Kiev {
+  application: string
+  environment: string
+
+  constructor (application: string, environment: string) {
+    this.application = application
+    this.environment = environment
+  }
+
+  /**
+   * Changes the current logger level.
+   *
+   * @param {LogLevel} level the log level.
+   */
+  public setLevel (level: LogLevel): void {
+    logger.setLevel(level)
   }
-}
 
-/**
- * Creates the message structure.
- *
- * @param message is a JSON object that represents the body of the event.
- * @param event  is the name of request event (request_started, request_finished and etc). The default is request_finished.
- * @param messageEvent is the name of the system event (BookingFound, BookingNotFound and etc). The default value is 'request'.
- * @param logLevel is the log level (info, warn, error).
- */
-// prettier-ignore
-function createMessage (
-  message: unknown,
-  event: string,
-  messageEvent: string,
-  logLevel: LogLevel
-): string | Message {
-  const data: Message = {
-    timestamp: new Date().toISOString(),
-    log_level: logLevel,
-    event: event,
-    body: message,
-    message: {
-      event: messageEvent
+  /**
+   * Returns the current logger level.
+   *
+   */
+  public getLevel () {
+    let level: LogLevel | null = null
+
+    for (const [name, number] of Object.entries(logger.levels)) {
+      if (logger.getLevel() === number) {
+        level = name as LogLevel
+        break
+      }
     }
+    return level
   }
 
-  return JSON.stringify(data)
-}
+  /**
+   * Generates an output log with severity level DEBUG.
+   *
+   * @param {String} message is the name of log event(i.e. request_started, request_finished and etc).
+   * @param {object} payload is a JSON object that represents the content of the event.
+   */
```

Comment:
[Suggestion]
The message field isn't really the _"log event(i.e. request_started, request_finished and etc)."_. It is the message, and might also be _the log event_.

```suggestion
   * @param {String} message is the core information of your log entry. By reading it anyone should understand what the log entry is about.
   * @param {object} payload is a JSON object with all structured fields to be added to the log.
   */
```